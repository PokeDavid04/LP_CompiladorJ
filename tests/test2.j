a =: 5

b =: (1 2 3)

a + b

c =: a + 2

c

d =: b * 2

d

e =: a + b * 3

e

f =: c ^ 2

f

g =: d | 2

g

b =: b + (3 2 1)

b

i =: _1

b * i
