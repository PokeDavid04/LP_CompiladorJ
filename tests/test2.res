a =: <asignada> 

b =: <asignada> 

6 7 8 

c =: <asignada> 

7 

d =: <asignada> 

2 4 6 

e =: <asignada> 

8 11 14 

f =: <asignada> 

49 

g =: <asignada> 

0 2 2 

b =: <asignada> 

4 4 4 

i =: <asignada> 

_4 _4 _4 

