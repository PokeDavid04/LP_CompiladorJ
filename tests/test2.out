a =: 5 

b =: 1 2 3 

6 7 8 

c =: 7 

7 

d =: 2 4 6 

2 4 6 

e =: 8 11 14 

8 11 14 

f =: 49 

49 

g =: 0 2 2 

0 2 2 

b =: 4 4 4 

4 4 4 

i =: _1 

_4 _4 _4 
